ST_resgroups.c

```diff
@@ -662,7 +663,7 @@ void rgroup_Establish(void) {
 
                 if (num_est) {
                     /* printf("%d %d %d %d\n",
-                     Globals.currIter, Globals.currYear, sp, num_est); */
+                     Globals->currIter, Globals->currYear, sp, num_est); */
```

Comment:
Can we get rid of this commented-out code?

Reply:
yes